pkg/reconciler/testing/v1/eventpolicy.go

```diff
@@ -44,25 +44,72 @@ func NewEventPolicy(name, namespace string, o ...EventPolicyOption) *v1alpha1.Ev
 	return ep
 }
 
-func WithInitEventPolicyConditions(et *v1alpha1.EventPolicy) {
-	et.Status.InitializeConditions()
+func WithInitEventPolicyConditions(ep *v1alpha1.EventPolicy) {
+	ep.Status.InitializeConditions()
 }
 
-func WithReadyEventPolicyCondition(ep *v1alpha1.EventPolicy) {
-	ep.Status.Conditions = []apis.Condition{
-		{
-			Type:   v1alpha1.EventPolicyConditionReady,
+func WithTrueAuthenticationEnabledCondition(ep *v1alpha1.EventPolicy) {
+	ep.Status.Conditions = append(ep.Status.Conditions,
+		apis.Condition{
+			Type:   v1alpha1.EventPolicyConditionAuthenticationEnabled,
 			Status: corev1.ConditionTrue,
-		},
+		})
+}
+
+func WithFalseAuthenticationEnabledCondition(ep *v1alpha1.EventPolicy) {
+	ep.Status.Conditions = append(ep.Status.Conditions,
+		apis.Condition{
+			Type:   v1alpha1.EventPolicyConditionAuthenticationEnabled,
+			Status: corev1.ConditionFalse,
+			Reason: "AuthOIDCFeatureNotEnabled",
+		})
+}
+
+func WithTrueSubjectsResolvedCondition(ep *v1alpha1.EventPolicy) {
```

Comment:
```suggestion
func WithEventPolicySubjectsResolvedSucceeded(ep *v1alpha1.EventPolicy) {
```

Reply:
Thanks, I updated it in the newer commit.